src/app/app.module.ts

```diff
@@ -5,20 +5,16 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { GuildsComponent } from './guilds/guilds.component';
-import { MaterialModule } from './shared/material.module';
 
 @NgModule({
   declarations: [
-    AppComponent,
-    GuildsComponent
+    AppComponent
   ],
   imports: [
     AppRoutingModule,
     BrowserAnimationsModule,
     BrowserModule,
-    HttpClientModule,
-    MaterialModule
```

Comment:
It could be, yes.  And I might prefer it that way.  The example I was following pulled Material out of the main module and added it to the feature modules.  So it's like: Main module imports feature modules --> Feature modles import material module.